src/allotropy/allotrope/schema_mappers/adm/plate_reader/benchling/_2023/_09/plate_reader.py

```diff
@@ -226,33 +226,7 @@ def map_model(self, data: Data) -> Model:
                     ASM_converter_version=ASM_CONVERTER_VERSION,
                 ),
                 plate_reader_document=[
-                    PlateReaderDocumentItem(
-                        analyst=data.metadata.analyst,
-                        measurement_aggregate_document=MeasurementAggregateDocument(
-                            analytical_method_identifier=measurement_group.analytical_method_identifier,
-                            experimental_data_identifier=measurement_group.experimental_data_identifier,
-                            container_type=ContainerType.well_plate,
-                            plate_well_count=TQuantityValueNumber(
-                                value=measurement_group.plate_well_count
-                            ),
-                            measurement_time=self.get_date_time(
-                                assert_not_none(
-                                    measurement_group.measurement_time
-                                    or data.metadata.measurement_time,
-                                    msg="Failed to parse valid timestamp",
-                                )
-                            ),
-                            measurement_document=[
-                                self._get_measurement_document_item(
-                                    measurement, data.metadata
-                                )
-                                for measurement in measurement_group.measurements
-                            ],
-                            processed_data_aggregate_document=self._get_processed_data_aggregate_document(
-                                measurement_group.processed_data
-                            ),
-                        ),
-                    )
+                    self._get_technique_document(measurement_group, data.metadata)
```

Comment:
Note to reviewer: changed in this file are just standardizing schema mapper code structure